tox.ini

```diff
@@ -1,9 +1,15 @@
 [tox]
-envlist = py26, py27, pypy, py31, py32, py33, py34, py35, docs, pep8, py3pep8
```

Comment:
because they don't have any `testenv` defined in `tox.ini` so they just rerun the tests